tools/patrist/patrist.py

```diff
@@ -0,0 +1,151 @@
+import argparse
```

Comment:
Can you convert this script to be compatible for python 3? Python 2.7 is being deprecated in January of this year.